add panel info for the auo panel in at least some samsung galaxy book go devices

before the patch there is a dmesg line like:
"panel-simple-dp-aux aux-ti_sn65dsi86.aux.0: Unknown panel AUO 0x203d, using conservative timings"

the base entry was just copied from a similar one
from that the 0x203d is taken and then the lst entry can be obtained via:

# apt-get install edid-decode
# edid-decode /sys/class/drm/card0-eDP-1/edid

in the output it can be found at the end at "Alphanumeric Data String"

...
    Alphanumeric Data String: 'AUO'
    Alphanumeric Data String: 'B140HTN02.0 '
...

diff --git a/drivers/gpu/drm/panel/panel-edp.c b/drivers/gpu/drm/panel/panel-edp.c
index 13bafed790f3..1488bedb4a5a 100644
--- a/drivers/gpu/drm/panel/panel-edp.c
+++ b/drivers/gpu/drm/panel/panel-edp.c
@@ -1881,6 +1881,7 @@ static const struct edp_panel_entry edp_panels[] = {
 	EDP_PANEL_ENTRY('A', 'U', 'O', 0x1062, &delay_200_500_e50, "B120XAN01.0"),
 	EDP_PANEL_ENTRY('A', 'U', 'O', 0x1e9b, &delay_200_500_e50, "B133UAN02.1"),
 	EDP_PANEL_ENTRY('A', 'U', 'O', 0x1ea5, &delay_200_500_e50, "B116XAK01.6"),
+	EDP_PANEL_ENTRY('A', 'U', 'O', 0x203d, &delay_200_500_e50, "B140HTN02.0"),
 	EDP_PANEL_ENTRY('A', 'U', 'O', 0x405c, &auo_b116xak01.delay, "B116XAK01"),
 	EDP_PANEL_ENTRY('A', 'U', 'O', 0x615c, &delay_200_500_e50, "B116XAN06.1"),
 	EDP_PANEL_ENTRY('A', 'U', 'O', 0x8594, &delay_200_500_e50, "B133UAN01.0"),
